src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Models/TestDto.cs

```diff
@@ -39,5 +50,36 @@ public class TestDto
         public string DiscountLogoPath { get; set; }
 
         public bool HasUserAnswered { get; set; }
+
+        private static bool SetIsAdvent(Microsoft.Extensions.Configuration.IConfiguration configuration)
+        {
+            var isAdventEndDate = configuration.GetSection("Competition:EndDate").Value;
+            var isAdventStartDate = configuration.GetSection("Competition:StartDate").Value;
+            ValidateResultPublicationDateTime(isAdventEndDate);
```

Comment:
logic with Validation should be in `https://github.com/DevAdventCalendar/DevAdventCalendar/blob/develop/src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Extensions/StartupExtensions.cs`